widgets/GoogleSubscriber-streamingsource.json

```diff
@@ -0,0 +1,65 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "Google Cloud PubSub",
+  "configuration-groups": [
+    {
+      "label" : "Basic",
+      "properties" : [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName",
+          "widget-attributes" : {
+            "placeholder": "Name used to identify this sink for lineage"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
+          "name": "project",
+          "widget-attributes" : {
+            "default": "auto-detect"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Subscriber Name",
+          "name": "subscriber",
+          "widget-attributes" : {
+            "placeholder": "Subscriber to subscribe to"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "PubSub topic",
+          "name": "topic",
+          "widget-attributes" : {
+            "placeholder": "Topic name for the subscriber"
+          }
+        }
+      ]
+    },
+    {
+      "label" : "Credentials",
+      "properties" : [
+        {
+          "widget-type": "textbox",
+          "label": "Service Account File Path",
+          "name": "serviceFilePath",
+          "widget-attributes" : {
+            "default": "auto-detect"
+          }
+        }
+      ]
+    }
+  ],
```

Comment:
there is some non-editable schema widget that will force the schema to some value in the  UI. Check the twitter source in hydrator-plugins for an example.